ll-time-input.html

```diff
@@ -84,10 +93,31 @@
      * @private
      */
     _isoToDisplay: function (isoStr) {
-      var m = moment(isoStr, 'HH:mm:ss');
-      if (m.isValid()) {
+      var m = this._getValidIsoMoment(isoStr);
+      if (!!m) {
```

Comment:
What are you trying to accomplish with this? I get that it is converting m to be a boolean, however I am not understanding why it is necessary. Can you help me understand what is happening?